.github/workflows/publish_to_pypi.yaml

```diff
@@ -0,0 +1,59 @@
+name: Publish Python distributions to PyPI and TestPyPI
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build-n-publish:
+    name: Build and publish to PyPI
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        # Adding -l {0} helps ensure conda can be found properly.
+        shell: bash -l {0}
+    env:
+      ENV_NAME: publish
+      PYTHON: 3.8
+    steps:
+      - uses: actions/checkout@main
+        with:
+          fetch-depth: 0
+
+      - name: Setup Mambaforge
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+            miniforge-variant: Mambaforge
+            miniforge-version: latest
+            use-mamba: true
+            python-version: ${{ env.PYTHON }}
+            environment-file: ci/${{ env.ENV_NAME }}.yaml
+            activate-environment: ${{ env.ENV_NAME }}
```

Comment:
can't you just use simple `setup-python` action here? Why do you need conda/mamba?